From cc01f7e0061a0f0a98d2586b651265c3746b785a Mon Sep 17 00:00:00 2001
From: Charalampos Stratakis <cstratak@redhat.com>
Date: Tue, 3 Feb 2026 03:04:41 +0100
Subject: [PATCH] Rename the benchmark file

The test_ prefix of the benchmark file was making
pytest collect it and use the benchmarks directory as
root, having the tests failing.

Rename the file so it's not collected via the usual
pytest runs.
---
 pythran/benchmarks/{test_benchmark.py => benchmark_runner.py} | 0
 1 file changed, 0 insertions(+), 0 deletions(-)
 rename pythran/benchmarks/{test_benchmark.py => benchmark_runner.py} (100%)

diff --git a/pythran/benchmarks/test_benchmark.py b/pythran/benchmarks/benchmark_runner.py
similarity index 100%
rename from pythran/benchmarks/test_benchmark.py
rename to pythran/benchmarks/benchmark_runner.py
-- 
2.52.0

